.circleci/config.yml

```diff
@@ -596,18 +599,12 @@ workflows:
           message: Pending Approval for Creation of Release Branch
           channel: 'pdt_releases'
           color: '#0E1111'
-      - build-all
```

Comment:
I decided to remove jobs that we weren't running as part of the scheduled build. I don't see these as being needed.